16/umbraco-cms/fundamentals/setup/upgrading/README.md

```diff
@@ -1,11 +1,15 @@
 ---
-description: This is the guide for upgrading existing Umbraco installations.
+description: Select the guide that fits your upgrade situation and learn more about the recommended approach.
 ---
 
 # Upgrade your project
 
 In this section, you will find everything you need to upgrade your Umbraco CMS project.
 
-If you are new to upgrades, see our introduction to [upgrades in Umbraco](./upgrade-introduction.md). This will give you an understanding of what goes on and the different topics you need to consider.
+Before upgrading, it is recommended to **read the introduction to [upgrades in Umbraco](./upgrade-introduction.md)**. This provides an understanding of what happens and the different topics you need to consider.
 
-Then we have step-by-step [upgrade instructions](./upgrade-details.md) for how to update your project to a newer version.
\ No newline at end of file
+## Upgrade Guides
+
+* [Upgrade details: Minor and Major upgrades](upgrade-details.md)
```

Comment:
The linked article title is **Upgrade Instructions**. Might make sense to change the link text to the article title so as not to confuse readers?